api.php

```diff
@@ -16,7 +16,7 @@
  * THIS IS AN ENTRY POINT
  */
 
-require_once('config.inc.php');
+require_once('includes/Startup.php');
```

Comment:
Does this change or a similar change need to be made to the `index.php` files used to prevent access to internal paths, like `/docker/index.php` and `/includes/index.php`? Trying to access `http://$baseurl/includes/`, for example, results in this:

![image](https://github.com/enwikipedia-acc/waca/assets/623117/02954985-6aef-4fa3-b6e2-5a769e23d14a)

The redirect works as expected in ae442df2f6c8ebe0c578e60391ce5b7ea766c0f7 from `multiproject/globalroles`. I suppose this also works, but is maybe not the optimal approach? Likewise, happy for this to be fixed in a follow-up PR, so not blocking this PR.